packages/editor/src/store/defaults.js

```diff
@@ -14,7 +14,11 @@ export const PREFERENCES_DEFAULTS = {
  *  allowedBlockTypes  boolean|Array Allowed block types
  *  richEditingEnabled boolean       Whether rich editing is enabled or not
  *  codeEditingEnabled boolean       Whether code editing is enabled or not
- *  enableCustomFields boolean       Whether the WordPress custom fields are enabled or not
+ *  enableCustomFields boolean       Whether the WordPress custom fields are enabled or not.
+ *                                     true  = the user has opted to show the Custom Fields panel at the bottom of the editor.
+ *                                     false = the user has opted to hide the Custom Fields panel at the bottom of the editor.
+ *                                     undefined = the current environment does not support Custom Fields,
```

Comment:
I'm not 100% on the location either, but this looks good to me. I'll do a little looking for prior art, and leave a note if I find anything helpful.

One idea on copy here -- I'm wondering if we can make it a little clearer what happens to the interface when this is set to `undefined`.

Maybe something like `the current environment does not support Custom Fields, so the option in Preferences -> Panels to enable the Custom Fields panel is not displayed.`?

I thought about "is hidden" as well, but since it's not rendered, wasn't sure if that's exactly right. What do you think?